build-static.sh

```diff
@@ -144,10 +150,10 @@ else
 		fi
 	fi
 
-	if [ "${SPC_REL_TYPE}" = "binary" ]; then
-		mkdir static-php-cli/
+	if [ "${SPC_REL_TYPE}" = "binary" ] && [[ ! "${arch}" =~ arm ]]; then
+		mkdir -p static-php-cli/
 		cd static-php-cli/
-		curl -o spc -fsSL "https://dl.static-php.dev/static-php-cli/spc-bin/nightly/spc-linux-$(uname -m)"
+		curl -o spc -fsSL "https://dl.static-php.dev/static-php-cli/spc-bin/nightly/spc-linux-${arch}"
```

Comment:
`spc` has arm binary, but it's using GNU-style arch name: `aarch64`, not `arm64`.

Reply:
I think this is still valid then. `arch` is set to `uname -m` at the top of the file, so we're just re-using the variable here. And if `arch` is set to `arm64`, then the download will fail since a binary with `arm64` does not exist